applications/StructuralMechanicsApplication/custom_elements/small_displacement_mixed_volumetric_strain_element.cpp

```diff
@@ -965,9 +908,29 @@ int  SmallDisplacementMixedVolumetricStrainElement::Check(const ProcessInfo& rCu
 {
     KRATOS_TRY
 
-    int base_element_check = SmallDisplacementMixedVolumetricStrainElement::BaseType::Check(rCurrentProcessInfo);
+    int check = SmallDisplacementMixedVolumetricStrainElement::BaseType::Check(rCurrentProcessInfo);
+
+    // Base check
+    check = StructuralMechanicsElementUtilities::BaseElementCheck(this, rCurrentProcessInfo);
```

Comment:
In here you check for the dynamic variables. Although they are not used right now, I'd keep the check for when we implement the dynamics of the element.

Reply:
The dynamic are only checked that are registered, even if the problem is static will not give any issue (would mean you are not importing the core :P)

Reply:
Yeah!